src/plugins/proxy/tests/batch_compliance_test.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/proxy/properties.hpp"
+#include "openvino/runtime/properties.hpp"
+#include "proxy_tests.hpp"
+
+using namespace ov::proxy::tests;
+
+TEST_F(ProxyTests, can_parse_and_inherit_batch_property) {
+    register_plugin_support_reshape(core,
+                                        "MOCK_DEVICE",
```

Comment:
added test.
ilya, @ilyachur  another question, if alias name not same as real device name, e.g register_plugin("real_name", "alias_name")
is_proxy_device(real_name) will report false in this case? because only alias name updated in registy:
pluginRegistry[alias] = desc;
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/dev/core_impl.cpp#L411C18-L411C18

Reply:
Proxy is a virtual plugin and we use only alias to operate with this plugin

Reply:
> Proxy is a virtual plugin and we use only alias to operate with this plugin

so if in future we have another plugin with device name = "VPU", alias name = "NPU" for example, we might have problem again?
because  is_proxy_plugin("VPU") will report false and the allow_auto_batch property will be erased again?

Reply:
@yangwang201911